src/main/webapp/i18n/zh-cn/eventAttendee.json

```diff
@@ -28,7 +28,7 @@
       },
       "userId": "用户编号",
       "eventId": "活动编号",
```

Comment:
can change line 29 and 30 to ID so all pages use same translation

Reply:
Resolved in 2d6dc9ee70121543ae0be8433fcfc8f9801146cc